packages/runtime/container-runtime/src/batchTracker.ts

```diff
@@ -36,41 +36,38 @@ export class BatchTracker {
 			this.trackedBatchCount++;
 		});
 
-		this.batchEventEmitter.on(
-			"batchEnd",
-			(error: any | undefined, message: BatchTrackerMessage) => {
-				assert(
-					this.startBatchSequenceNumber !== undefined &&
-						this.batchProcessingStartTimeStamp !== undefined,
-					0x2ba /* "batchBegin must fire before batchEnd" */,
-				);
+		this.batchEventEmitter.on("batchEnd", (error: unknown, message: BatchTrackerMessage) => {
```

Comment:
This file is purely indentation changes because `any | undefined` -> `unknown` reduced the length of this line enough for everything to align differently.